### ISSUE TITLE
Project is not being saved correctly

### ISSUE THREAD

**Issue by akurzhan** [ucbtrans/opt#2]
1. Saved scenarios have only links and nodes. Other expected data include models, road parameters, commodities, demand, split ratios. 
2. Settings: what units are used?
3. We agreed before that OPT project will be wholly described by a single XML file, which will have .opt extension. Currently, the project config references external scenario files.

**Comment by ggomes** [comment]
**1.     Saved scenarios have only links and nodes. Other expected data include models, road parameters, commodities, demand, split ratios.**
I am going to put together a slide with the complete scenario tree for you to refer to. The scenario files themselves (scenarioA.xml and scenarioB.xml) have all of these items. Think of the items as falling into three categories: 
models = {models}
supply = {network}
demand = {commodities, demands, split ratios}
Up until now I have only implemented the OPT/OTM interface for the "supply" category, which means network (ie road segments) and road parameters. 
I can implement the demands interface very easily. But the first task is to display the geometry. 

**2.     Settings: what units are used?**
Which settings are you asking about? The interface for a segment is here:
https://github.com/ucbtrans/opt/blob/master/src/main/java/opt/data/Segment.java

**3.     We agreed before that OPT project will be wholly described by a single XML file, which will have .opt extension. Currently, the project config references external scenario files.**
Yes. I was seeing it would take a little more doing to include the Maven provided schema for OTM in the schema for OPT. That is to extend a Maven-provided schema. So I decided to separate them temporarily. The ultimate goal is to have a single OPT file. I will open this as a separate "issue".
